### DISCUSSION TITLE
Fan Control reads but does not control fans when resuming from standby

### DISCUSSION THREAD

**Discussion by floepie05** [Rem0o/FanControl.Releases#2152] [Q&A]
Hi, recently fan control is not resuming from standby as expected.  Fan speeds are read correctly it seems, but the curves are not applied.

**Comment by Rem0o** [comment]
How do you identify the curves not applying? How does it manifests?

**Comment by floepie05** [comment]
Sorry, I meant that the fans are entirely no longer controllable when system wakes from standby.  The fan speeds all move to about 60% speed (1200/2000 RPM) when awoken.   None of the fans can be controlled manually either.  A restart of Fan Control solves it immediately.   The executable and json files are stored in a subdirectory within the 'Documents' folder.   

This issue has arisen only a "couple" weeks ago.  In total I have been using FC for about 10 months and never an issue!  :)

**Comment by floepie05** [comment]
So, I have been troubleshooting this further.  I downloaded a fresh "install".  Didn't help, same issue persists.  Restarting app OR "refresh sensors detections" (ctrl R) **always works** to fix the problem.

All fans are connected to a Lian Li controller.  The controller is synced to a the motherboard via PWM and speed sensor to the System Fan #3 header.  The plugin is copied into the plugins directory.

*What works* - If I control all fans in Fan Control using only the System Fan 3 sensor and control, there is no issue with resuming from standby.  But, controlling only one fan header does not offer much separate control. 

Normally I would find the plugin faulty.  However, in this case, a simple *Ctrl R* fixes the issue.  So, somewhere, Fan Control is not refreshing all exposed sensors (exposed by plugin) when the system resumes from standby.
